### DISCUSSION TITLE
Can I use Pycharm IDE for continueing this course?

### DISCUSSION THREAD

**Discussion by Umut9263** [mrdbourke/pytorch-deep-learning#309] [Q&A] [answered]
I didn't like Google Colab due to Uİ and slow autocomplete so I want to use pycharm with Cuda. Does anything change if I use pycharm?

**Comment by udayzee05** [comment, marked as answer]
Use vs code instead of PyCharm, you can use jupyter notebook in it as well, in PyCharm community version you won't able to view jupyter notebook and execute them vs code will be handy.
